File: engine/model/src/main/scala/net/server/channel/packet/DeleteTeleportRockMapPacket.scala
package net.server.channel.packet

class DeleteTeleportRockMapPacket(private var _type: Byte, private var _vip: Boolean, private var _mapId: Int) extends BaseTeleportRockMapPacket(_type = _type, _vip = _vip) {
  def mapId: Int = _mapId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/DeleteTeleportRockMapPacket.java
package net.server.channel.packet;

import net.server.MaplePacket;

public class DeleteTeleportRockMapPacket extends BaseTeleportRockMapPacket implements MaplePacket {
   private final Integer mapId;

   public DeleteTeleportRockMapPacket(Byte theType, Boolean vip, Integer mapId) {
      super(theType, vip);
      this.mapId = mapId;
   }

   public Integer mapId() {
      return mapId;
   }
}
